<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoTab</class>
 <widget class="QWidget" name="PhotoTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
     <item>
      <widget class="QLineEdit" name="photoLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhUrlCharactersOnly</set>
       </property>
       <property name="placeholderText">
        <string>Choose the image you want to crop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="photoButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="icon">
        <iconset resource="../resources2.qrc">
         <normaloff>:/resources/icons/picture.svg</normaloff>:/resources/icons/picture.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1f2c33</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="0,20,0,0,0">
        <item>
         <widget class="QCheckBox" name="toggleCheckBox">
          <property name="styleSheet">
           <string notr="true">QCheckBox:unchecked{color: red}
QCheckBox:checked{color: white}
QCheckBox::indicator{
	width: 20px;
	height: 20px;
}
QCheckBox::indicator:checked{
	image: url(:/resources/icons/checkbox_checked.svg);
}
QCheckBox::indicator:unchecked{
	image: url(:/resources/icons/checkbox_unchecked.svg);
}
QCheckBox::indicator:checked:hover{
	image: url(:/resources/icons/checkbox_checked_hover.svg);
}
QCheckBox::indicator:unchecked:hover{
	image: url(:/resources/icons/checkbox_unchecked_hover.svg);
}</string>
          </property>
          <property name="text">
           <string>Toggle Settings</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="mfaceCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox:unchecked{color: red}
QCheckBox:checked{color: white}
QCheckBox::indicator{
	width: 20px;
	height: 20px;
}
QCheckBox::indicator:checked{
	image: url(:/resources/icons/checkbox_checked.svg);
}
QCheckBox::indicator:unchecked{
	image: url(:/resources/icons/checkbox_unchecked.svg);
}
QCheckBox::indicator:checked:hover{
	image: url(:/resources/icons/checkbox_checked_hover.svg);
}
QCheckBox::indicator:unchecked:hover{
	image: url(:/resources/icons/checkbox_unchecked_hover.svg);
}</string>
          </property>
          <property name="text">
           <string>Multi-Face</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tiltCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox:unchecked{color: red}
QCheckBox:checked{color: white}
QCheckBox::indicator{
	width: 20px;
	height: 20px;
}
QCheckBox::indicator:checked{
	image: url(:/resources/icons/checkbox_checked.svg);
}
QCheckBox::indicator:unchecked{
	image: url(:/resources/icons/checkbox_unchecked.svg);
}
QCheckBox::indicator:checked:hover{
	image: url(:/resources/icons/checkbox_checked_hover.svg);
}
QCheckBox::indicator:unchecked:hover{
	image: url(:/resources/icons/checkbox_unchecked_hover.svg);
}</string>
          </property>
          <property name="text">
           <string>Autotilt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="exposureCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox:unchecked{color: red}
QCheckBox:checked{color: white}
QCheckBox::indicator{
	width: 20px;
	height: 20px;
}
QCheckBox::indicator:checked{
	image: url(:/resources/icons/checkbox_checked.svg);
}
QCheckBox::indicator:unchecked{
	image: url(:/resources/icons/checkbox_unchecked.svg);
}
QCheckBox::indicator:checked:hover{
	image: url(:/resources/icons/checkbox_checked_hover.svg);
}
QCheckBox::indicator:unchecked:hover{
	image: url(:/resources/icons/checkbox_unchecked_hover.svg);
}</string>
          </property>
          <property name="text">
           <string>Autocorrect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="imageWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
         <item>
          <widget class="QWidget" name="controlWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cropButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources2.qrc">
          <normaloff>:/resources/icons/crop.svg</normaloff>:/resources/icons/crop.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
     <item>
      <widget class="QLineEdit" name="destinationLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhUrlCharactersOnly</set>
       </property>
       <property name="placeholderText">
        <string>Choose where you want to save the cropped image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="destinationButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Destination Folder</string>
       </property>
       <property name="icon">
        <iconset resource="../resources2.qrc">
         <normaloff>:/resources/icons/folder.svg</normaloff>:/resources/icons/folder.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toggleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>controlWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
